drools-ha/ha-core-infra/src/test/resources/logback-test.xml

```diff
@@ -28,14 +28,18 @@
     <producerConfig>bootstrap.servers=localhost:9092</producerConfig>
   </appender>
 
-  <logger name="org.hacep" level="warn" additivity="false">
+  <logger name="org.hacep" level="debug" additivity="false">
```

Comment:
@danielezonca and myself were against this to go in final merge, so hopefully this  only for testing the change here and will be removed before approved PR, is that correct?

Reply:
yes